packages/sdk/src/waku/waku.ts

```diff
@@ -224,6 +232,18 @@ export class WakuNode implements IWaku {
     return this.connectionManager.isConnected();
   }
 
+  private async dialMultiaddr(
```

Comment:
let's not introduce this operation and use `ConnectionManager`

Reply:
83acb84670debada7b0155c49926d63bbcc9858e